src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleObject = {};
+
+  const lines = sourceString
```

Comment:
The variable `sourceString` should be checked to ensure it is a string before proceeding with string operations. Consider adding a type check to handle cases where `sourceString` might not be a string.